src/libraries/Common/src/Interop/Unix/System.Net.Security.Native/Interop.Initialization.cs

```diff
@@ -24,11 +24,6 @@ static SslInitializer()
 
             //Call ssl specific initializer
             Ssl.EnsureLibSslInitialized();
-            if (Interop.Crypto.ErrPeekLastError() != 0)
-            {
-                // It is going to be wrapped in a type load exception but will have the error information
-                throw Interop.Crypto.CreateOpenSslCryptographicException();
```

Comment:
Should we clean up the queue and put the error to trace log for troubleshooting?